x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/header_info/sentinel_one/header_sentinel_one_info.tsx

```diff
@@ -20,7 +22,14 @@ interface HeaderSentinelOneInfoProps {
 
 export const HeaderSentinelOneInfo = memo<HeaderSentinelOneInfoProps>(
   ({ agentId, platform, hostName }) => {
-    const { data } = useGetSentinelOneAgentStatus([agentId]);
+    const isSentinelOneV1Enabled = useIsExperimentalFeatureEnabled(
+      'responseActionsSentinelOneV1Enabled'
+    );
+
+    const { data } = useAgentStatus([agentId], 'sentinel_one', {
+      refetchInterval: DEFAULT_POLL_INTERVAL,
```

Comment:
is this hardcoded on purpose? I mean, in endpoint we used `autoRefresh`flag too.